CONTRIBUTING.md

```diff
@@ -68,7 +68,7 @@ $ ./bin/start_editor
 ```
 
 In each case, a browser window will be opened to the container running Swagger UI.  
```

Comment:
I'm going to file a follow up issue for correcting this documentation. Since our schema is now split into multiple files, we probably shouldn't be referring to just the main file.